components/breadcrumb/stories/breadcrumb.mdx

```diff
@@ -0,0 +1,64 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as BreadcrumbStories from './breadcrumb.stories';
+
+<Meta of={BreadcrumbStories} title="Docs" />
+
+<Title of={BreadcrumbStories} />
+<Description of={BreadcrumbStories} />
+
+## Variants
+
+### Standard
+
+<Canvas of={BreadcrumbStories.Default} />
+
+### Dragged
+
+<Canvas of={BreadcrumbStories.Dragged} />
+
+### Nested
+
+<Canvas of={BreadcrumbStories.Nested} />
+
+### Nested (root visible)
+
+<Canvas of={BreadcrumbStories.NestedRootVisible} />
+
+### Multiline
+
+<Canvas of={BreadcrumbStories.Multiline} />
+
+### Multiline (dragged)
+
+<Canvas of={BreadcrumbStories.MultilineDragged} />
+
+### Multiline Nested
+
+<Canvas of={BreadcrumbStories.NestedMultiline} />
+
+### Multiline Nested (root visible)
+
+<Canvas of={BreadcrumbStories.MultilineNestedRootVisible} />
+
+### Compact
+
+<Canvas of={BreadcrumbStories.Compact} />
+
+### Compact Nested
+
+<Canvas of={BreadcrumbStories.CompactNested} />
+
+### Compact Nested (root visible)
+
+<Canvas of={BreadcrumbStories.CompactNestedRootVisible} />
+
+### Disabled
+
+The example below has two disabled breadcrumb items. When disabling the text link, the `is-disabled` class gets added to `.spectrum-Breadcrumbs-itemLink`. When disabling the Action button, the `[disabled]` attribute is applied.
+
+<Canvas of={BreadcrumbStories.Disabled} />
+
+## Properties
```

Comment:
```suggestion
## Properties

The component accepts the following inputs (properties):
```

From the .storybook/DocumentationTemplate.mdx template that adds a nice context.

Reply:
Oh great idea. I might make a card to go back and double check some of the other recently-merged components and make sure they have this as well.

Reply:
should be added in this commit: https://github.com/adobe/spectrum-css/pull/2793/commits/56a86b9a361961f2ee6757764347e16f3523d730

Reply:
@castastrophe Should we name this heading "Story arguments", "Story args", "Story arg types" or something like that, instead of "Properties"? I'm wondering if "Properties" if the best naming for what these are and what Storybook calls them. This would apply to our auto docs template and custom MDX files.

Reply:
Let's leave properties for now. Yes, we're expressing them in storybook as args but consumers reading the docs shouldn't need to understand how storybook works to understand what it is.  In this case, this is documenting the design API.